user/src/main/java/pl/gucio/enzo/chronica/user/data/dto/AccountDto.java

```diff
@@ -0,0 +1,13 @@
+package pl.gucio.enzo.chronica.user.data.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class AccountDto {
```

Comment:
Dto może być recordem, jest to idealny feature pod dtosa

Reply:
//done